wqflask/base/data_set.py

```diff
@@ -49,6 +49,7 @@
 import codecs
 
 import json
+import ijson
```

Comment:
*issue:* Submit a patch for `python-ijson` to me or @arunisaac in our channel before we merge, otherwise this will cause gn2 to fail...